Cargo.toml

```diff
@@ -1,4 +1,65 @@
+[package]
+name = "bob_management"
+description = "Bob Management GUI"
+publish = false
+keywords = [ "BOB", "Management", "GUI" ]
+version.workspace = true
+authors.workspace = true
+license-file.workspace = true
+edition.workspace = true
+readme.workspace = true
+repository.workspace = true
+
+[workspace.package]
+version = "0.0.0"   
+authors = ["Romanov Simeon ArchArcheoss@proton.me"]
+repository = "https://github.com/qoollo/bob-management"
+readme = "./README.md"
+license-file = "./LICENSE"
+edition = "2021"
+
+[dependencies]
+cli = { path = "./cli", optional = true }
+backend = { path = "./backend", optional = true }
+frontend = { path = "./frontend", optional = true }
+
 [workspace]
-members = [
-    "bob-management-server",
-]
\ No newline at end of file
+members = [ "cli", "backend", "frontend", "./" ]
+default-members = [ "backend", "./", "frontend" ]
+
+[features]
+default = [ "swagger", "backend", "frontend" ] 
+backend = [ ]
+frontend = [ "dep:frontend" ]
+swagger = [ "backend/swagger" ]
+dummy = [ ]   # HACK: Do not turn on
+
+[profile.dev]
+debug-assertions = true
+
+[profile.release]
+# Optimize for size
+# opt-level = "s"
+# Optimize for speed
+opt-level = 3 
+
+# Slightly increase perfomance and reduce binary size
+panic = "abort"
+
+[profile.release-lto]
+inherits = "release"
+# Link Time optimization, causes a bit longer compilation
+lto = true
+# Maximize size reduction optimization, causes longer compilation
+codegen-units = 1 
+
+[profile.min-size]
+inherits = "release"
+opt-level = "s"
+
+# HACK
+[[bin]]
```

Comment:
You can try virtual workspace like I wrote here: https://github.com/qoollo/bob-management/pull/9#discussion_r1344233749